### ISSUE TITLE
Accessibility & Rich Editor: Alt text for images in posts

### ISSUE THREAD

**Issue by matthewbienz** [vanilla/vanilla#4295]
WYSIWYG needs an input field for a description

Add ability to add alt text through the WYSIWYG editor to the images posted in _my_ discussions so that visually impaired users using assistive technology and search engines can read the images.

**Comment by beckyvb** [comment]
I'm looking into the feasibility of this. Simply adding the alt text to the upload dropdown doesn't work, since the dropdown closes once an upload occurs. If the dropdown stayed open (which would be a little clunky, IMO), the user could add alt text for the upload but they would have no way to edit the alt text. Further, they may add alt text to a file upload which would just do nothing. This isn't great either.

The only way I can see this functionality working is to add "edit image" and "delete image" icons on the image when editing a post. The edit image menu could also include width and height adjustments.

It may be worth noting that currently, a user can add alt text to an image by editing the HTML.

**Comment by slafleche** [comment]
This will be possible in the new editor.
